kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -1075,6 +1181,25 @@ class ResidualBasedNewtonRaphsonStrategy
             << "ATTENTION: max iterations ( " << mMaxIterationNumber
             << " ) exceeded!" << std::endl;
     }
+    
+    /**
+     * @brief This method returns the default settings
+     */
+    virtual Parameters GetDefaultSettings()
+    {
+        Parameters default_settings(R"({
+            "max_iterations" : 30
```

Comment:
you should also have defsults for compute_reactions etc

Reply:
ok!